diracx-db/src/diracx/db/sql/sandbox_metadata/db.py

```diff
@@ -50,11 +54,9 @@ def get_pfn(bucket_name: str, user: UserInfo, sandbox_info: SandboxInfo) -> str:
         return "/" + "/".join(parts)
 
     async def insert_sandbox(
-        self, se_name: str, user: UserInfo, pfn: str, size: int
+        self, owner_id: int, se_name: str, pfn: str, size: int
     ) -> None:
         """Add a new sandbox in SandboxMetadataDB."""
-        # TODO: Follow https://github.com/DIRACGrid/diracx/issues/49
```

Comment:
The linked issue is not solved. It does not apply here anymore?

Reply:
The linked issue is not solved:
- I break down the method into smaller methods in `diracx-db` to move the logic from it to `diracx-logic`.
- The comment was moved to `diracx-logic` because it makes more sense to read it from here now.